### PR TITLE
init

### PR DESCRIPTION
**Pull request by StepShad0w**

### CODE REVIEW

File: src/App.jsx [hunk 1]
```diff
@@ -2,86 +2,54 @@ import React, { useState } from "react";
...
+          <h1>Input your card</h1>
+          <input
+            type="text"
+            placeholder="Enter card value"
+            value={cardValue}
+            onChange={(e) => setCardValue(e.target.value)}
+          />
+          <select value={suit} onChange={handleSuitChange}>
+            <option value="">Select suit</option>
+            {Object.entries(suits).map(([key, value]) => (
```
Comment (SlavikMelnyk) [inline src/App.jsx]: Should we use Object.entries here?

File: src/components/CardSuits.js [hunk 2]
```diff
@@ -0,0 +1,13 @@
+export const suits = {
```
Comment (SlavikMelnyk) [inline src/components/CardSuits.js]: we should use naming conventions here,
https://www.syncfusion.com/blogs/post/top-javascript-naming-convention

File: src/components/CardSuits.js [hunk 3]
```diff
@@ -0,0 +1,13 @@
...
+    "♥️": "Hearts",
+    "♦️": "Diamonds",
+    "♣️": "Clubs",
+    "♠️": "Spades",
+  };
+export const colors={
```
Comment (SlavikMelnyk) [inline src/components/CardSuits.js]: the same here

File: src/App.jsx [hunk 4]
```diff
@@ -2,86 +2,54 @@ import React, { useState } from "react";
 import { ImSpades } from "react-icons/im";
 import { PiHeartStraightFill } from "react-icons/pi";
 import "./App.css";
+import { cardValues } from "./components/CardValues";
```
Comment (SlavikMelnyk) [inline src/App.jsx]: Also, would be good if you move all constants to seperate constants.js file, for ex: src/helpers/constants.js

File: src/components/CardSuits.js (hunk 2 above)
Comment (StepShad0w) [reply src/components/CardSuits.js]: replaced the names of constants

File: src/components/CardSuits.js (hunk 3 above)
Comment (StepShad0w) [reply src/components/CardSuits.js]: here also repleced

Comment (StepShad0w) [reply src/components/CardSuits.js]: here also replaced

File: src/App.jsx (hunk 4 above)
Comment (StepShad0w) [reply src/App.jsx]: created a folder "utils" and in it a file for constants

File: src/App.jsx (hunk 1 above)
Comment (StepShad0w) [reply src/App.jsx]: we can use Object.keys but then we can't see the suit names

Comment (SlavikMelnyk) [reply src/App.jsx]: Got it

File: src/components/CardSuits.js (hunk 2 above)
Comment (SlavikMelnyk) [reply src/components/CardSuits.js]: push the changes)

**Pull request merged**